Dockerfile-arm

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017, 2019 the Velero contributors.
```

Comment:
Copyright boilerplate should have 2020 as the year.

Reply:
Year has been updated.